apps/meteor/client/components/PlanTag.tsx

```diff
@@ -1,32 +1,30 @@
 import { Box, Tag } from '@rocket.chat/fuselage';
 import { useSafely } from '@rocket.chat/fuselage-hooks';
-import { useMethod } from '@rocket.chat/ui-contexts';
+import { useEndpoint } from '@rocket.chat/ui-contexts';
 import React, { ReactElement, useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
 
 import { ILicenseTag } from '../../ee/app/license/definitions/ILicenseTag';
 
+type PlanTagType = {
+	name: string;
+	color: string;
+};
+
 function PlanTag(): ReactElement {
-	const [plans, setPlans] = useSafely(
-		useState<
-			{
-				name: string;
-				color: string;
-			}[]
-		>([]),
-	);
+	const [plans, setPlans] = useSafely(useState<PlanTagType[]>([]));
 
-	const getTags = useMethod('license:getTags');
+	const isEnterpriseEdition = useEndpoint('GET', '/v1/licenses.isEnterprise');
+	const { data } = useQuery(['licenses.isEnterprise'], () => isEnterpriseEdition(), {
+		refetchOnWindowFocus: false,
+	});
 
 	useEffect(() => {
 		const developmentTag = process.env.NODE_ENV === 'development' ? { name: 'development', color: '#095ad2' } : null;
+		const enterpriseTag = data?.isEnterprise ? { name: 'enterprise', color: '#095ad2' } : null;
```

Comment:
Is this color `primary-600`? Maybe use a reference instead of a hardcoded value.